pyscriptjs/examples/fractals.py

```diff
@@ -96,7 +94,7 @@ def newton(
     xr: Range = (-2.5, 1),
     yr: Range = (-1, 1),
     max_iterations: int = 100
-) -> (np.array, np.array):
+) -> tuple[np.array, np.array]:
```

Comment:
@verhulstm For example, `flake8` tested the code and discovered a Python SyntaxError.